packages/core/src/browser/shell/tab-bars.ts

```diff
@@ -143,9 +143,9 @@ export class TabBarRenderer extends TabBar.Renderer {
      * @param {boolean} isInSidePanel An optional check which determines if the tab is in the side-panel.
      * @returns {VirtualElement} The virtual element of the rendered tab.
      */
-    override renderTab(data: SideBarRenderData, isInSidePanel?: boolean): VirtualElement {
+    override renderTab(data: SideBarRenderData, isInSidePanel?: boolean, isPartOfHiddenTabBar?: boolean): VirtualElement {
```

Comment:
Do you mind updating the typedocs for the method to include the new parameter `isPartOfHiddenTabBar`?

Reply:
Oops I missed that, will add it in 👍🏽